app.json

```diff
@@ -2,9 +2,20 @@
   "name": "api.mozillascience.org",
   "description": "API server for science.mozilla.org",
   "addons": [
-    "heroku-postgresql"
+    "heroku-postgresql:hobby-dev"
   ],
   "scripts": {
-    "postdeploy": "python app/manage.py migrate"
+    "postdeploy": "sh scripts/postdeploy.sh"
+  },
+  "env": {
+    "DJANGO_ENV": "review",
```

Comment:
@gideonthomas I dont think so I created an application which deployed automatcally on each master push, and the posdeploy script did not run there.